tests/test_plan_refsols/access_partition_child_backref_calc.txt

```diff
@@ -1,6 +1,5 @@
 ROOT(columns=[('part_name', name), ('part_type', part_type_1), ('retail_price_versus_avg', retail_price - avg_price)], orderings=[])
  JOIN(conditions=[t0.part_type == t1.part_type], types=['inner'], columns={'avg_price': t0.avg_price, 'name': t1.name, 'part_type_1': t1.part_type, 'retail_price': t1.retail_price})
-  PROJECT(columns={'avg_price': agg_0, 'part_type': part_type})
-   AGGREGATE(keys={'part_type': part_type}, aggregations={'agg_0': AVG(retail_price)})
-    SCAN(table=tpch.PART, columns={'part_type': p_type, 'retail_price': p_retailprice})
+  AGGREGATE(keys={'part_type': part_type}, aggregations={'avg_price': AVG(retail_price)})
```

Comment:
What triggered moving the projection (`'avg_price': agg_0, `) to aggregation ? 
I can see how this is obvious and correct behavior but wonder what part of the code in this PR triggered it.